php/WP_CLI/Autoloader.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace WP_CLI;
+
+/**
+ * Class Autoloader.
+ *
+ * This is a custom autoloader to replace the functionality that we would
```

Comment:
This doc would be more helpful if it explained why the autoloader existed, in addition to what it does